Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -656,9 +354,18 @@ void ProcessInput ()
 					ProcessWinChange ();
 				}
 				if (wch == Curses.KeyMouse) {
-					Curses.getmouse (out Curses.MouseEvent ev);
-					//System.Diagnostics.Debug.WriteLine ($"ButtonState: {ev.ButtonState}; ID: {ev.ID}; X: {ev.X}; Y: {ev.Y}; Z: {ev.Z}");
-					mouseHandler (ToDriverMouse (ev));
+					int wch2 = wch;
+
+					while (wch2 == Curses.KeyMouse) {
+						KeyEvent key = null;
+						ConsoleKeyInfo [] cki = new ConsoleKeyInfo [] {
+							new ConsoleKeyInfo ('\u001b', 0, false, false, false),
```

Comment:
Please introduce a named constant for '\u001b' to make the code more readable. 

E.g.

```cs
const char ESC = '\u001b`;
```

Reply:
Done by replacing with with  `(char)Key.Esc`.